configure.ac

```
@@ -427,6 +433,10 @@ adl_RECURSIVE_EVAL([$luadir], fluxluadir)
 AS_VAR_SET(fluxluadir, $fluxluadir)
 AC_SUBST(fluxluadir)
 
+AS_VAR_SET(fluxbindingincludedir, $includedir/flux/_binding)
```

To disambiguate from our other bindings, it might be worth doing the following:
```suggestion
AS_VAR_SET(fluxbindingincludedir, $includedir/flux/_py_binding)
```

---

I'd be fine with that, but was wondering if it would make sense to leave it ambiguous since you could use the cleaned up headers with luajit ffi, for example, just as well as python cffi.  This is more indicative of how we use them, but there's no intrinsic python-ness involved in the headers.

---

> was wondering if it would make sense to leave it ambiguous since you could use the cleaned up headers with luajit ffi, for example, just as well as python cffi. 

Ah! That makes a lot of sense.  Sounds good to leave it as is then.